openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -91,14 +92,10 @@
   .openassessment_student_info_form {
     margin-bottom: ($baseline-v/2);
 
-    .list--actions__item {
-      @include text-align(left);
-
-      .action--submit-username {
-        @extend %btn--secondary;
+    .action--submit-username {
+      @extend %btn--primary;
 
-        margin-left: 0;
-      }
+      margin-left: 0;
```

Comment:
This isn't your change, but this isn't RTL-compatible. It should use:

    @include margin-left(0);